components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/docs/examples/alerts/alerts_config.yaml

```diff
@@ -0,0 +1,114 @@
+get:
+  - id: GetAlertConfigurations
+    request:
+      method: GET
+      url: https://localhost:9443/api/am/publisher/v1.0/alerts/AbnormalRequestTime/configurations
+      headers: |
+        Authorization: Bearer eff01a97-4dc0-3542-a184-75aecf355ec2
+    response:
+      status:
+        code: 200
+        msg: OK
+      body:
+        {
+          "count": 1,
+          "list": [
+          {
+            "configurationId": "UGl6emFTaGFja0FQSSMxLjAuMA",
+            "configuration": {
+              "apiName": "PizzaShackAPI",
+              "apiVersion": "1.0.0",
+              "requestTime": "120"
+            }
+          }
+          ]
+        }
+      headers: |
+        Content-Type: application/json
+  - rawRequest: |
+      GET https://localhost:9443/api/am/publisher/v1.0/alerts/AbnormalRequestTime/configurations/UGl6emFTaGFja0FQSSMxLjAuMA HTTP/1.1
+      Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+    curl: |
+      curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8" https://localhost:9443/api/am/publisher/v1.0/alerts/AbnormalRequestTime/configurations/UGl6emFTaGFja0FQSSMxLjAuMA
+    rawResponse: |
+      {
+        "count": 1,
+        "list": [
+          {
+            "configurationId": "UGl6emFTaGFja0FQSSMxLjAuMA",
+            "configuration": {
+              "apiName": "PizzaShackAPI",
+              "apiVersion": "1.0.0",
+              "requestTime": "120"
+            }
+          }
+        ]
+      }
+
+put:
+  - id: AddAlertConfiguration
+    request:
+      method: PUT
+      url: https://localhost:9443/api/am/publisher/v1.0/alerts/AbnormalRequestTime/configurations/UGl6emFTaGFja0FQSSMxLjAuMA
+      headers: |
+        Authorization: Bearer eff01a97-4dc0-3542-a184-75aecf355ec2
+        Content-Type: application/json
+      body:
+        {
+          "requestTime":"120",
+          "apiName":"PizzaShackAPI",
+          "apiVersion":" 2.0"
+        }
+    response:
+      status:
+        code: 200
+        msg: OK
+      headers: |
+          Content-Type: application/json
+      body:
+        {
+          "configurationId":"UGl6emFTaGFja0FQSSMxLjAuMA",
+          "configuration":
+            {
+              "requestTime":"120",
+              "apiName":"PizzaShackAPI",
+              "apiVersion":" 2.0",
+            }
+        }
+
+  - rawRequest: |
```

Comment:
We can use 'rawRequest' only when the requests and responses are not easily structured to headers, parsable body etc. We do not need to set the same example in both formats. Shall we remove this?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117